### DISCUSSION TITLE
Skybox from cube map

### DISCUSSION THREAD

**Discussion by Preston-Harrison** [bevyengine/bevy#16518] [Q&A]
Is there any way to easily generate a skybox from a cube map (such as the one here https://opengameart.org/content/sky-box-sunny-day). The current example to generate a skybox is a little hard to follow.

**Comment by agjini** [comment]
Hi @Preston-Harrison,

We've also spent some time looking for information on how to set up a `Skybox` and an `EnvironmentLight`.

We came across this repository: [bevy_skybox_cli](https://github.com/bytestring-net/bevy_skybox_cli), which has been incredibly helpful to us!

We've successfully integrated a skybox and an environment light (for material reflections) into our game using an HDRi image sourced from websites like PolyHaven.

Here is our [GitHub project](https://github.com/agjini/crustace/blob/main/pong/src/plugin/playground.rs).

You can see how we generate textures from the downloaded HDRi image in our brief [Readme.md](https://github.com/agjini/crustace/blob/main/pong/Readme.md).

I hope this helps you!

I wish the bevy_skybox_cli could be integrated as a plugin in a game to generate on the fly the assets (As suggested by the author)...

**Reply by agjini** [reply]
Here is a screenshot :-) 

![image](https://github.com/user-attachments/assets/98985dcc-c4a0-4b60-aa5d-f4270ee47ce8)

**Reply by Preston-Harrison** [reply]
Thanks for the heads up about the project! Very useful for me
